# Represents a maliput::api::IsoLaneVelocity.
float64 sigma_v
float64 rho_v
float64 eta_v
